data/sql/updates/pending_db_world/rev_1586845118268975400.sql

```diff
@@ -0,0 +1,94 @@
+(@HOLIDAYID, @DATEID+2, 379945024), -- 2022-11-23.
+(@HOLIDAYID, @DATEID+3, 396722240), -- 2023-11-23.
+(@HOLIDAYID, @DATEID+4, 413483072), -- 2024-11-22.
+(@HOLIDAYID, @DATEID+5, 430260288); -- 2025-11-22.
+
+-- Children's Week. Holiday ID: 201.
+UPDATE `game_event` SET `start_time`='2020-05-01 00:01:00', `occurence`='525600', `length`='10080' WHERE `eventEntry`=10;
+SET @HOLIDAYID := 201;
+SET @DATEID := 16;
+DELETE FROM `holiday_dates` WHERE `id`=@HOLIDAYID AND `date_id` IN (@DATEID, @DATEID+1, @DATEID+2, @DATEID+3, @DATEID+4, @DATEID+5);
+INSERT INTO `holiday_dates` (`id`, `date_id`, `date_value`) VALUES -- DONE.
+(@HOLIDAYID, @DATEID, 	339738624), -- 2020-05-01.
+(@HOLIDAYID, @DATEID+1, 356515840), -- 2021-05-01.
+(@HOLIDAYID, @DATEID+2, 373293056), -- 2022-05-01.
+(@HOLIDAYID, @DATEID+3, 390070272), -- 2023-05-01.
+(@HOLIDAYID, @DATEID+4, 406274048), -- 2024-04-30.
+(@HOLIDAYID, @DATEID+5, 423051264); -- 2025-04-30.
+
+-- Noblegarden. Holiday ID: 181.
+UPDATE `game_event` SET `start_time`='2020-04-13 00:01:00', `occurence`='525600', `length`='10080' WHERE  `eventEntry`=9;
+SET @HOLIDAYID := 181;
+SET @DATEID := 13;
+DELETE FROM `holiday_dates` WHERE `id`=@HOLIDAYID AND `date_id` IN (@DATEID, @DATEID+1, @DATEID+2, @DATEID+3, @DATEID+4, @DATEID+5);
+INSERT INTO `holiday_dates` (`id`, `date_id`, `date_value`) VALUES -- DONE.
+(@HOLIDAYID, @DATEID, 	338886656), -- 2020-04-13.
+(@HOLIDAYID, @DATEID+1, 355663872), -- 2021-04-13.
+(@HOLIDAYID, @DATEID+2, 372441088), -- 2022-04-13.
+(@HOLIDAYID, @DATEID+3, 389218304), -- 2023-04-13.
+(@HOLIDAYID, @DATEID+4, 405979136), -- 2024-04-12.
+(@HOLIDAYID, @DATEID+5, 422756352); -- 2025-04-12.
+
+-- Harvest Festival. Holiday ID: 321.
+UPDATE `game_event` SET `start_time`='2020-09-29 00:01:00', `occurence`='525600', `length`='10080'  WHERE  `eventEntry`=11;
+SET @HOLIDAYID := 321;
+SET @DATEID := 14;
+DELETE FROM `holiday_dates` WHERE `id`=@HOLIDAYID AND `date_id` IN (@DATEID, @DATEID+1, @DATEID+2, @DATEID+3, @DATEID+4, @DATEID+5);
+INSERT INTO `holiday_dates` (`id`, `date_id`, `date_value`) VALUES -- DONE.
+(@HOLIDAYID, @DATEID, 	344391680), -- 2020-09-29.
+(@HOLIDAYID, @DATEID+1, 361168896), -- 2021-09-29.
+(@HOLIDAYID, @DATEID+2, 377946112), -- 2022-09-29.
+(@HOLIDAYID, @DATEID+3, 394723328), -- 2023-09-29.
+(@HOLIDAYID, @DATEID+4, 411484160), -- 2024-09-28.
+(@HOLIDAYID, @DATEID+5, 428261376); -- 2025-09-28.
+
+-- Lunar Festival
+UPDATE `game_event` SET `length` = 20160 WHERE `eventEntry` = 7;
+UPDATE `holiday_dates` SET `holiday_duration`=336 WHERE `id`=327;
+
+-- End Dates
+
+UPDATE `game_event` SET `end_time` = '2030-12-31 06:00:00' WHERE `eventEntry` IN (3,  73,  81,  79,  53,  54,  77,  70,  80,  71,  65,  34,  35,  64,  63,  62,  43,  51,  36,  37,  69,  68,  67,  45,  44,  42,  41,  40,  39,  38,  52,  26,  27,  25,  24,  11,  12,  23,  21,  20,  19,  18,  14,  10,  9,  1,  2,  4,  30,  5,  15,  6,  29,  7,  8,  28,  16,  50,  72,  75,  74,  33,  76,  32,  78);
```

Comment:
We don't need to set End Time. delete end_time lines.

https://github.com/azerothcore/azerothcore-wotlk/blob/c839ce53988c544052ae96a82f325fe441ad1747/src/server/game/Events/GameEventMgr.cpp#L244

Reply:
@Kitzunu